package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ahryman40k/ts-fhir-types",
-  "version": "4.0.29",
```

Comment:
Because library is FHIR R4 standard, it will be nice to maintain 4.0.x format.

Reply:
No problem. I committed the suggested changes, so now it stays at 4.0.x